villain/types.rkt

```diff
@@ -11,6 +11,8 @@
 (define type-port       #b101)
 (define type-vector     #b110)
 (define type-flonum     #b111)
+(define type-proc       #b111)
```

Comment:
This conflicts with the `type-flonum` tag.

Reply:
Yes, thanks!  Will use higher bits.